duckduckgo/Android
# apply white menu

Asana Issue URL: https://app.asana.com/0/inbox/392891325557408/531181521679102/531510529165908

## Description

Use a white menu instead of the jarring orange/red one.

Also fix bug where the menu doesn't disappear when tapping outside of it (on API 22 or lower).

Tested on API 22, 23 and 26 (emulator) and One Plus One (API 22).
 
## Steps to Test this PR:
1. Launch the app - open the menu. 
1. Check Back/forward look (and are) disabled.
1. Check Bookmarks button looks (and is) enabled.
1. Search for something and click a link.  
1. Open menu and back button should look enabled.
1. Tap back - should navigate back. 
1. Open menu - forward should look and be enabled.
1. Tap forward - should navigate forward
1. Open menu - tap bookmarks - should open bookmarks UI
